docs/section6/Rule6-11.md

```diff
@@ -0,0 +1,46 @@
+
+# Lighting - Rule 6-11  
+
+**Rule ID:** 6-11   
+**Rule Description:**Where retail display lighting is included in the proposed building design the display lighting additional power shall be less than or equal to the limits established by Section 9.5.2.2(b)  
+**Rule Assertion:** Proposed RMD = expected value  
+**Appendix G Section:** Table 9.5.2.2
```

Comment:
I recommend including the Appendix G section that triggers this requirement as I think this is a semi-grey area in Appendix G and reading this without the App G reference is a little mind boggling as to why this would be required for the proposed. I recall discussing it previously, based on my memory, I think we all decided that it is required because of Section G1.2.1b.1 because it says "Tables G3.7-1 and G3.7-2 **and the methodology described in Section 9.5.2**"